ext/oci8/oci8.stub.php

```diff
@@ -2,534 +2,468 @@
 
 /** @generate-function-entries */
 
-/** @param resource $statement_resource */
-function oci_define_by_name($statement_resource, string $column_name, mixed &$variable, int $type = 0): bool {}
+/** @param resource $statement */
+function oci_define_by_name($statement, string $column, mixed &$result, int $type = 0): bool {}
 
 /**
- * @param resource $statement_resource
+ * @param resource $statement
  * @alias oci_define_by_name
  * @deprecated
  */
-function ocidefinebyname($statement_resource, string $column_name, mixed &$variable, int $type = 0): bool {}
+function ocidefinebyname($statement, string $column, mixed &$result, int $type = 0): bool {}
 
-/** @param resource $statement_resource */
-function oci_bind_by_name($statement_resource, string $column_name, mixed &$variable, int $maximum_length = -1, int $type = 0): bool {}
+/** @param resource $statement */
+function oci_bind_by_name($statement, string $param, mixed &$var, int $max_length = -1, int $type = 0): bool {}
 
 /**
- * @param resource $statement_resource
+ * @param resource $statement
  * @alias oci_bind_by_name
  * @deprecated
  */
-function ocibindbyname($statement_resource, string $column_name, mixed &$variable, int $maximum_length = -1, int $type = 0): bool {}
+function ocibindbyname($statement, string $param, mixed &$var, int $max_length = -1, int $type = 0): bool {}
 
-/** @param resource $statement_resource */
-function oci_bind_array_by_name($statement_resource, string $column_name, mixed &$variable, int $maximum_array_length, int $maximum_item_length = -1, int $type = SQLT_AFC): bool {}
+/** @param resource $statement */
+function oci_bind_array_by_name($statement, string $param, mixed &$var, int $max_array_length, int $max_item_length = -1, int $type = SQLT_AFC): bool {}
```

Comment:
I'm not sure `$max_array_length` and `$max_item_length` should be changed to `$array_length` and `$item_length`, so I'm leaving them for now.